sda-commons-server-hibernate/src/main/java/org/sdase/commons/server/hibernate/HibernateBundle.java

```diff
@@ -93,7 +93,6 @@ public void initialize(Bootstrap<?> bootstrap) {
   @Override
   public void run(C configuration, Environment environment) throws Exception {
     delegate.run(configuration, environment);
-    environment.jersey().register(this);
```

Comment:
Why was it registered before? Is it necessary for "UnitOfWork", i.e. transactions to work properly?
Might be a change for users of the Hibernate bundle if they now need to register the bundle themselves.

Reply:
It was there from the beginning without further explanation. We register components at Jersey to be able to use `@Context` in the class or expose endpoints. Is there any other reason why a class or an instance should be registered? We use the `io.dropwizard.hibernate.HibernateBundle` which will register health checks and a `UnitOfWorkApplicationListener` for transaction management.

Reply:
> Is there any other reason why a class or an instance should be registered? 

Can't say. We hardly use that bundle.

Reply:
https://www.dropwizard.io/en/latest/manual/hibernate.html

Why do they register DAOs with Jersey?

Reply:
They register the resource there. Not the DAO.

Reply:
True. 🤦